lib/ansible/plugins/connection/psrp.py

```diff
@@ -207,6 +209,15 @@ def __init__(self, *args, **kwargs):
         self._shell_type = 'powershell'
         super(Connection, self).__init__(*args, **kwargs)
 
+        if C.DEFAULT_DEBUG:
```

Comment:
I'm just wondering whether we need to explicit set to DEBUG when DEFAULT_DEBUG is enabled considering that's what it does right now. We should probably just set  to INFO when not C.DEFAULT_DEBUG.

Reply:
Yeah, I am not sure if DEBUG will stay the default, and this would be more safe :-)
But I don't mind implementing your proposal, really. It's not my problem to fix.

Reply:
Fixed.